plotting/plotIt/MCFiles.yml

```diff
@@ -22,11 +22,46 @@
   legend: 'TT Full Lept'
   order: 6
 
-#'TT_TuneCUETP8M1_13TeV-powheg-pythia8_MiniAODv2_v1.0.0+7415-10-gcf61438_ZAAnalysis_dd80b32_histos.root':
-#  type: mc
-#  cross-section: 831.76
-#  generated-events: 19757200.0
-#  fill-color: '#EB6841'
-#  legend: 'TT incl'
-#  order: 6
+'ST_tW_antitop_5f_inclusiveDecays_13TeV-powheg_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 35.85
+  generated-events: 1  #988500
+  order: 5
+  group: 'ST'
+
+'ST_tW_top_5f_inclusiveDecays_13TeV-powheg_Fall15MiniAODv2_v1.2.0+7415-73-g91bfcf0_ZAAnalysis_4f7ac83_histos.root':
+  type: mc
+  cross-section: 35.85
```

Comment:
I had 35.6 for both... from https://twiki.cern.ch/twiki/bin/view/CMS/SummaryTable1G25ns#Single_top

where are yours from ?